project/SparkBuild.scala

```diff
@@ -51,6 +51,11 @@ object BuildCommons {
   // Root project.
   val spark = ProjectRef(buildLocation, "spark")
   val sparkHome = buildLocation
+
+  val testTempDir = s"$sparkHome/target/tmp"
+  if (!new File(testTempDir).isDirectory()) {
+    require(new File(testTempDir).mkdirs())
+  }
```

Comment:
Yeah, sorry I would have been more clear.  This was a general comment about how we are doing things in the SBT build and not something that should have blocked merging this PR.